pkgs/data/themes/catppuccin-sddm-corners/default.nix

```diff
@@ -14,8 +15,14 @@ stdenvNoCC.mkDerivation {
     hash = "sha256-sTnt8RarNXz3RmYfmx4rD+nMlY8rr2n0EN3ntPzOurw=";
   };
 
+  # Might also need qt-svg, qt-quickcontrols2
```

Comment:
if we're going to include the qt dependencies in this derivation. could you add those